src/Student.java

```diff
@@ -0,0 +1,58 @@
+import java.lang.invoke.SwitchPoint;
+
+public class Student {
+    private String name;
+    private  Characteristics[] characteristics;
+    double magicPower;
+    double transgressionDistance;
+
+
+
+
+
+    public Student(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Characteristics[] getCharacteristics() {
+        return characteristics;
+    }
+
+    public void setCharacteristics(Characteristics[] characteristics) {
+        this.characteristics = characteristics;
+    }
+
+    public double getMagicPower() {
+        return magicPower;
+    }
+
+    public void setMagicPower(double magicPower) {
+        this.magicPower = magicPower;
+    }
+
+    public double getTransgressionDistance() {
+        return transgressionDistance;
+    }
+
+    public void setTransgressionDistance(double transgressionDistance) {
+        this.transgressionDistance = transgressionDistance;
+    }
+
+    public void PrintCharacteristics(Student student){
```

Comment:
Имена методов должны быть с маленькой буквы
```suggestion
    public void printCharacteristics(Student student){
```